dev/wlp-mavenRepoTasks/src/com/ibm/ws/wlp/mavenFeatures/utils/Constants.java

```diff
@@ -55,8 +55,20 @@
 	public static final String LICENSE_URL_MAVEN = "http://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/maven/licenses/L-JTHS-8SZMHX/HTML/";
 	public static final String LICENSE_COMMENTS_MAVEN = "Additional notices http://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/maven/licenses/L-JTHS-8SZMHX/HTML/notices.html";
 	
+	public static final String WEBSPHERE_LIBERTY_BOM = "WebSphere Liberty features bill of materials";
+	public static final String WEBSPHERE_LIBERTY_JSON = "WebSphere Liberty features JSON";
+	public static final String OPEN_LIBERTY_BOM = "Open Liberty features bill of materials";
+	public static final String OPEN_LIBERTY_JSON = "Open Liberty features JSON";
 
-
+	public static final String OPEN_LIBERTY_SCM_CONNECTION = "scm:git:git@github.com:OpenLiberty/open-liberty.git";
+	public static final String OPEN_LIBRETY_SCM_URL = "git@github.com:OpenLiberty/open-liberty.git";
+	public static final String OPEN_LIBERTY_SCM_TAG = "HEAD";
+	public static final String OPEN_LIBERTY_URL = "https://openliberty.io/";
+	
+	public static final String ERICGLAU_DEV_ID = "ericglau";
```

Comment:
Make the constant names more generic such as DEV_ID, since the developer could change.